src/add/observable/bindCallback.ts

```diff
@@ -3,4 +3,8 @@ import {BoundCallbackObservable} from '../../observable/BoundCallbackObservable'
 
 Observable.bindCallback = BoundCallbackObservable.create;
 
-export var _void: void;
\ No newline at end of file
+declare module '../../Observable' {
+  module Observable {
```

Comment:
mainly I'm just used to using module for so long.  I have no issues changing it to `namespace`.